web-registry/src/pages/Dashboard/MyEcocredits/MyEcocredits.tsx

```diff
@@ -305,7 +305,7 @@ export const MyEcocredits = (): JSX.Element => {
                       ),
                       label: CREDIT_SEND_TITLE,
                       onClick: () => {
-                        track('send1');
+                        track<'send1'>('send1');
```

Comment:
ditto

Reply:
same here, i have a commit with this that i haven't pushed yet